### PR TITLE
Support vellum case management migration via UI

### PR DESCRIPTION
**Pull request by orangejenny**
@benrudolph / @emord 

Would be nice to get deployed soon so Kevin can start migrating the early adopters (who thought they were going to be migrated a week or two ago).

### CODE REVIEW

File: corehq/apps/hqadmin/views.py [hunk 1]
```diff
@@ -491,26 +492,47 @@ def post(self, request, *args, **kwargs):
...
+                    text_content=text_content,
+                    email_from=settings.SUPPORT_EMAIL)
+                m.emailed = datetime.now()
+                m.save()
+                successes.add(d)
+            elif action == 'migrate':
+                for app_id in get_app_ids_in_domain(d):
+                    try:
+                        management.call_command('migrate_app_to_cmitfb', app_id)
+                    except Exception as x:
```
Comment (dimagimon) [inline corehq/apps/hqadmin/views.py]: F841 local variable 'x' is assigned to but never used

File: corehq/apps/hqadmin/views.py [hunk 2]
```diff
@@ -491,26 +492,47 @@ def post(self, request, *args, **kwargs):
...
+                        m.notes = m.notes + "{}failed on app {}".format('; ' if m.notes else '', app_id)
+                        errors.add(d)
+                if d not in errors:
+                    successes.add(d)
+                m.migrated = datetime.now()
+                m.save()
+        if len(successes):
+            messages.success(request, "Succeeded with the following {} domains: {}".format(
+                                        len(successes), ", ".join(successes)))
```
Comment (dimagimon) [inline corehq/apps/hqadmin/views.py]: E126 continuation line over-indented for hanging indent

File: corehq/apps/hqadmin/views.py [hunk 3]
```diff
@@ -491,26 +492,47 @@ def post(self, request, *args, **kwargs):
...
+        if len(errors):
+            messages.error(request, "Errors in the following {} domains: {}".format(
+                                        len(errors), ", ".join(errors)))
```
Comment (dimagimon) [inline corehq/apps/hqadmin/views.py]: E126 continuation line over-indented for hanging indent

**Comment by emord** [comment]
@orangejenny I'm assuming there aren't any tests for this code. Looks good to merge if that's the case

File: corehq/apps/hqadmin/views.py [hunk 4]
```diff
@@ -491,26 +492,47 @@ def post(self, request, *args, **kwargs):
...
-                    m.emailed = datetime.now()
-                    m.save()
-        return json_response({'status': 'success'})
+            return json_response({'success': 'success'})
+
+        domains = self.request.POST['domains'].split(",")
+        errors = set([])
+        successes = set([])
+        for d in domains:
+            m = VCMMigration.objects.get(domain=d)
```
Comment (millerdev) [inline corehq/apps/hqadmin/views.py]: nitpick: single letter variable names are not ideal

**Comment by nickpell** [comment]
>  I'm assuming there aren't any tests for this code. Looks good to merge if that's the case

merging

**Comment by nickpell** [comment]
leaving branch so tests get run anyway

File: corehq/apps/hqadmin/views.py (hunk 4 above)
Comment (orangejenny) [inline corehq/apps/hqadmin/views.py]: fair, will fix in next PR

**Pull request merged**